### PR TITLE
qa/tests: added rgw into upgrade sequence to improve coverage

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: qa/suites/upgrade/nautilus-x/parallel/3-upgrade-sequence/upgrade-mon-osd-mds.yaml [hunk 1]
```diff
@@ -45,3 +46,8 @@ upgrade-sequence:
        wait-for-healthy: true
    - sleep:
        duration: 60
+   - ceph.restart:
+       daemons: [client.1]
```
Comment (cbodley) [inline qa/suites/upgrade/nautilus-x/parallel/3-upgrade-sequence/upgrade-mon-osd-mds.yaml]: upgrade-all.yaml should also restart client.1, probably along with `daemons: [mds.a]`

Comment (yuriw) [reply qa/suites/upgrade/nautilus-x/parallel/3-upgrade-sequence/upgrade-mon-osd-mds.yaml]: you are right, added

**Comment by yuriw** [comment]
http://pulpito.front.sepia.ceph.com:80/yuriw-2019-07-23_16:54:09-upgrade:nautilus-x-master-distro-basic-smithi/

File: qa/suites/upgrade/nautilus-x/parallel/1-ceph-install/nautilus.yaml [hunk 2]
```diff
@@ -42,3 +42,6 @@ tasks:
 - install.upgrade:
     client.0:
 - print: "**** done install.upgrade on client.0"
+- rgw:
+   - client.1
+- print: "**** done started rgw client.1"
```
Comment (cbodley) [inline qa/suites/upgrade/nautilus-x/parallel/1-ceph-install/nautilus.yaml]: i think this chunk needs to come before `parallel:` above

**Pull request merged**